docker/iml-warp-drive.dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM rust:1.36 as builder
+FROM rust:nightly as builder
 WORKDIR /build
 COPY . .
 RUN cargo build -p iml-warp-drive --release
 
-FROM rust:1.36
+FROM rust:nightly
```

Comment:
Should these be beta and not nightly?

Reply:
No beta tag is available for rust docker images. This will have to be updated to 1.37 once it's released.